tests/README.md

```diff
@@ -36,6 +36,10 @@ TC_PATHS="tests/kafkatest/tests/client/pluggable_test.py::PluggableConsumerTest"
 ```
 TC_PATHS="tests/kafkatest/tests/client/pluggable_test.py::PluggableConsumerTest.test_start_stop" bash tests/docker/run_tests.sh
 ```
+* Run a specific test method with specific parameters
+```
+TC_PATHS="tests/kafkatest/tests/streams/streams_upgrade_test.py::StreamsUpgradeTest.test_metadata_upgrade" _DUCKTAPE_OPTIONS='--parameters '\''{"from_version":"0.10.1.1","to_version":"2.6.0-SNAPSHOT"}'\' bash tests/docker/run_tests.sh
```

Comment:
Great example. Wondering if it is correct? 
```
--parameters '\''
```

Should this be
```
--parameters '\'
```

because that the end there is `'\'`

Reply:
Haha, having just spent several hours settling on this exact sequence of characters, I can certify that it's correct ;) 

Actually, I would like to get at least one other person to try it out. Do you mind?

To answer your question, here's how it works:
first string (wrapped in single quotes for consistency with the second string): `--parameters `
first escaped single quote: `\'`
second string (wrapped in single quotes to avoid evaluating the special characters): `{"from_version":"0.10.1.1","to_version":"2.6.0-SNAPSHOT"}`
second escaped single quote: `\'`

Following Bash rules, a sequence of strings and characters just get concatenated, so all of the above gets smashed together to form the desired final string:
`--parameters '{"from_version":"0.10.1.1","to_version":"2.6.0-SNAPSHOT"}'`

which will then be evaluated _again_ by Bash inside the docker container, so that the final argument to ducktape is the desired argument list: `--parameters`, followed by `{"from_version":"0.10.1.1","to_version":"2.6.0-SNAPSHOT"}`.

Reply:
I tried it out on Friday but left it running and forgot to report back. It passed for me 👍